### ISSUE TITLE
create global.model and global.view

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/tandem#96]
`Tandem.globalTandem` was recently added for global elements.  What is the convention for the structure of this element subtree?  Should it be organized into `global.model` and `global.view`, like screens?   

For example, Gas Properties (and other sims) have model and view separation of things in `PhET Menu > Options`.   

Should I organize like this? (my preferred choice)

```
global.model.projectorModeEnabledProperty
global.model.pressureNoiseProperty 
global.view.optionsDialog
global.view.optionsDialog.projectorModeCheckbox
global.view.optionsDialog.pressureNoiseCheckbox
```

Or like this? (current implementation in GP)

```
global.projectorModeEnabledProperty
global.pressureNoiseProperty 
global.optionsDialog
global.optionsDialog.projectorModeCheckbox
global.optionsDialog.pressureNoiseCheckbox
```

Or something else?

**Comment by pixelzoom** [comment]
Hmmm... Looks like the OptionsDialog has to be under `general...phetMenu.optionsDialog`.  How the do I get the dialog content to appear under there?

**Comment by zepumph** [comment]
I think `global.view` and `global.model` make sense are are idiomatic to the setup of other screens.  @samreid @chrisklus can you think of a reason that we wouldn't want to make this change?

**Comment by samreid** [comment]
Sounds great to me!

**Comment by samreid** [comment]
I wasn't sure who should implement this, so I invited both @pixelzoom and @zepumph

**Comment by pixelzoom** [comment]
Before implementing, shouldn't a designer be consulted?

**Comment by samreid** [comment]
@kathy-phet or @arouinfar can you please review https://github.com/phetsims/tandem/issues/96#issue-469462699 and comment?

**Comment by chrisklus** [comment]
Sounds good to me as well.

**Comment by arouinfar** [comment]
`global.model` and `global.view` sounds reasonable to me.

**Comment by kathy-phet** [comment]
Where does this appear in the tree? Is it under "general" or is it a new main header? 
We have "sim.general. .... " and the "sim.screenname...." is this "sim.global.model..."?

**Comment by zepumph** [comment]
We have 3 things allowed on the sim root:

* `general`, which is stuff that pertains to all sims
* `global`, which is sim specific stuff that pertains to all screens
* `xScreen` which pertains only to that screen.

I think the goal of this issue is to determin if all three of these should be further sub divided with a `view` and `model` structure. Currently only the screens are.

**Comment by kathy-phet** [comment]
OK. I see. I think having model and view for the "global" tandems consistent with how the screen work is good. 

Are you proposing also to change the "general" tandems?

**Comment by zepumph** [comment]
>Are you proposing also to change the "general" tandems?

It wasn't explicit in this issue, but I don't see why we would do it for one but not the other.

**Comment by kathy-phet** [comment]
I'm wondering for general, if it is going to make it harder to find things? Right now we have a "Navigation Bar" there which makes it easy to find the stuff in the navigation bar. But if that was first model and then view it would be harder to find. That is my only concern with this path - the instructional designer looking for stuff. What would it look like in "general" if it were to change?

**Comment by zepumph** [comment]
Let's discuss conventions for `general` over in https://github.com/phetsims/tandem/issues/108.

**Comment by zepumph** [comment]
>. I think having model and view for the "global" tandems consistent with how the screen work is good.

It sounds like `global.model` and `global.view` are approved by designers. @pixelzoom do you think that we should create these as statics on `Tandem`, like `Tandem.GLOBAL_MODEL` and `Tandem.GLOBAL_VIEW` and in fact hide `Tandem.GLOBAL` from public consumption?

**Comment by pixelzoom** [comment]
> do you think that we should create these as statics on `Tandem` ...

Yes, I guess so. Is there another (better?) alternative?

> ... like `Tandem.GLOBAL_MODEL` and `Tandem.GLOBAL_VIEW` and in fact hide `Tandem.GLOBAL` from public consumption?

If we want this to be "consistent with how the screens work", then there may be things as the top-level that are above both `global.model` and `global.view`.  So we'll probably want all 3 of those tandems.

**Comment by zepumph** [comment]
@pixelzoom please review. I named them as was decided in https://github.com/phetsims/tandem/issues/95.

**Comment by pixelzoom** [comment]
`Tandem.GLOBAL_MODEL` and `Tandem.GLOBAL_VIEW` look good.   But I still see `Tandem.globalTandem`.  Is that being addressed elsewhere, like maybe in #95?

**Comment by zepumph** [comment]
Yes that's right, I was trying to get something to work in projectile motion so I just added these, the rest will be converted in https://github.com/phetsims/tandem/issues/95.
